sdk/template/template/karma.conf.js

```diff
@@ -20,17 +27,18 @@ module.exports = function(config) {
       "karma-env-preprocessor",
       "karma-coverage",
       "karma-remap-istanbul",
-      "karma-junit-reporter"
+      "karma-junit-reporter",
+      "karma-json-to-file-reporter",
+      "karma-json-preprocessor"
```

Comment:
Same as above.
(`karma-json-to-file-reporter` is for the `jsonToFileReporter`, if that is being removed, this plugin can be removed.)
(`karma-json-preprocessor` is to preprocess the JSON files, look at `preprocessors` below, if the JSON file input is being removed, this plugin can be removed)